### PR TITLE
feat(web): Complete resort page

### PR DESCRIPTION
**Pull request by mooyg**
Completed the resort page renamed CategoryTab -> ResortCard

### CODE REVIEW

File: packages/api/src/index.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { chalkLog } from '@lib/chalkLog';
 
 const RedisStore = connectRedis(expressSession);
 
-const redisClient = createClient(process.env.OASIS_API_REDIS_URL);
+const redisClient = createClient({url:process.env.OASIS_API_REDIS_URL});
```
Comment (HenryLeC) [inline packages/api/src/index.ts:20]: Please don't change things unrelated to the pr. It just makes it so we can git blame and rollback changes without unintended side effects

File: packages/web/src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -8,13 +8,17 @@ import {
   Topics,
   Profile as ProfileIcon,
   Logout as LogoutIcon,
+} from '@components/icons';
+=======
```
Comment (HenryLeC) [inline packages/web/src/components/navbar/Navbar.tsx:12]: Please fix these merge lines

File: packages/web/src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -8,13 +8,17 @@ import {
...
 } from '@icons/index';
 import React, {useRef, useState } from 'react';
 import { Button } from '@components/common/Button';
 import { NavItem } from '@components/navbar/NavItem';
 import { DropdownItem } from '@components/navbar/DropdownItem';
 import { useGetCurrentUser } from '@lib/common/getCurrentUser';
 import { useRouter } from 'next/router';
+import {useOnClickOutside} from "@hooks/useOnClickOutside"
+=======
```
Comment (HenryLeC) [inline packages/web/src/components/navbar/Navbar.tsx:21]: Same with this one

File: packages/web/src/hooks/useOnClickOutside.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    }
+    if (ref.current) {
+      document.addEventListener("mousedown", listener);
+    }
+    return () => {
+ document.removeEventListener("mousedown", listener);
+    }
+  },[ref,handler])
+}
+
```
Comment (HenryLeC) [inline packages/web/src/hooks/useOnClickOutside.tsx:17]: This was already added in a previous commit

File: packages/web/tailwind.config.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  mode: 'jit',
```
Comment (HenryLeC) [inline packages/web/tailwind.config.js:2]: Why did you remove jit? @heybereket did you approve this?

Comment (heybereket) [reply packages/web/tailwind.config.js:2]: > Why did you remove jit? @heybereket did you approve this?

A day ago yes, but it worked fine, so we decided to keep it.
